Dockerfile.local

```diff
@@ -25,6 +25,6 @@ COPY hs2-loader/bin/schema-load /usr/local/bin/
 
 WORKDIR /opt/hive
 
-EXPOSE 10000
+EXPOSE 8000 10000 10002
 
-CMD java -jar hs2-launcher.jar
+CMD java -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=8000 -jar hs2-launcher.jar
```

Comment:
It is a very good idea to provide the option for connecting the debugger but doing it always may slow things down. I don't mind much since the image is not for production use but it might be easy to control it via an environment var or something similar. WDYT?

Reply:
At first I wanted to go exactly that way, but since we have the dual setup `Docker` vs `Docker.local`, I thought we could enable it by default in the local mode. 

Since local image is meant for dev or local debugging, I think it would be OK to have it on at all time.